src/apps/geoserver/wps/src/main/java/org/geoserver/cloud/wps/WpsApplicationConfiguration.java

```diff
@@ -12,8 +12,8 @@
 @ImportResource( //
         reader = FilteringXmlBeanDefinitionReader.class, //
         locations = { //
-            "jar:gs-wps-.*!/applicationContext.xml" // , //
-            ,
+            "jar:gs-wps-.*!/applicationContext.xml", //
+            "jar:gs-wfs-.*!/applicationContext.xml", //
```

Comment:
I'm not sure about this, but we can revisit later. Thing is it'll load all the beans defined in the gs-wfs applicationContext.xml, which we try to avoid, and hence the usage of regular expressions to whitelist or blacklist beans by name.
The trick is to identify which WFS beans will WPS require and which not.

Reply:
Probably we will need to test based each WPS process one-by-one. I detected that for those related to vectorial calculations (ie. gs:Clip) it was looking for some beans (then I realized that as in the case of WCS, then it was required to add the applicationContext.xml file).

Reply:
Honestly, I don´t know enough about WPS to know how much of WFS it would use. As a starter, it´d be enough if you can´t make HTTP WFS requests to the WPS service

Reply:
That is a good point, just to avoid misuse of WPS containers.